src/Uno.UI/DirectUI/BudgetManager.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-
+#if false
```

Comment:
@jeromelaban Everything in this file is unused. Can I just delete it?

Reply:
Let's keep it for now, we may need it as a reference to bring related code from WinUI.